api/client/stack/deploy.go

```diff
@@ -168,6 +168,10 @@ func deployServices(
 					Command: service.Command,
 					Args:    service.Args,
 					Env:     service.Env,
+					// Service Labels will not be copied to Containers
+					// automatically during the deployment so we apply
+					// it here.
+					Labels: getStackLabels(namespace, service.Labels),
```

Comment:
Thanks @dnephin for pointing that out. The pull request has been updated. Let me know if there are any issues.